### PR TITLE
added kick force.

### PR DESCRIPTION
**Pull request by sim0504**
Added kick force which can be different in real life and simulation.

### CODE REVIEW

File: Engine/Communication/sender/grsim_command_sender.py [hunk 1]
```diff
@@ -49,3 +50,12 @@ def _create_protobuf_packet(packets_frame) -> grSim_Packet.grSim_Packet:
             grsim_command.kickspeedz = 0
 
         return grsim_packet
+
+    @staticmethod
+    def translate_kick_force(kick_force: KickForce) -> int:
+        kick_translation = {KickForce.NONE: 0,
+                            KickForce.LOW: 1,
+                            KickForce.MEDIUM: 3,
+                            KickForce.HIGH: 5}
```
Comment (PhiBabin) [inline Engine/Communication/sender/grsim_command_sender.py]: Should be much lower I think. Please test it in simulation

File: Engine/Communication/sender/serial_command_sender.py [hunk 2]
```diff
@@ -19,10 +20,18 @@ def send_packet(self, packets_frame):
...
 
             if packet.charge_kick:
                 self.connection.charge(packet.robot_id)
+
+    @staticmethod
+    def translate_kick_force(kick_force: KickForce) -> int:
+        kick_translation = {KickForce.NONE: 0,
+                            KickForce.LOW: 1,
+                            KickForce.MEDIUM: 3,
+                            KickForce.HIGH: 5}
```
Comment (PhiBabin) [inline Engine/Communication/sender/serial_command_sender.py]: Need tests IRL

File: Util/ai_command.py [hunk 3]
```diff
@@ -39,7 +40,7 @@ def addMoveTo(self, target: [Pose, Position], cruise_speed=1, end_speed=0, ball_
         self._pathfinder_on = True
         return self
 
-    def addKick(self, kick_force=1):
+    def addKick(self, kick_force:KickForce=KickForce.LOW):
         self._kick_force = kick_force
```
Comment (PhiBabin) [inline Util/ai_command.py]: Assert if we pass a KickForce?

File: Engine/Communication/sender/grsim_command_sender.py (hunk 1 above)
Comment (sim0504) [reply Engine/Communication/sender/grsim_command_sender.py]: It looked fine. But we can tweak them

File: Util/ai_command.py (hunk 3 above)
Comment (sim0504) [reply Util/ai_command.py]: Yup, would be useful for the people who are unaware of the change

**Pull request merged**